dds/src/implementation/actors/data_writer_actor.rs

```diff
@@ -1027,10 +1003,14 @@ impl DataWriterActor {
         &mut self,
         data_writer_address: ActorAddress<DataWriterActor>,
         publisher: PublisherAsync,
-        offered_incompatible_qos_publisher_listener: Option<ActorAddress<PublisherListenerActor>>,
-        offered_incompatible_qos_participant_listener: Option<
+        (publisher_listener, publisher_listener_mask): (

```

Comment:
does that need to be tuples?

Reply:
We should remove the tuples later since they were already there for now.